lib/hexpm/accounts/organizations.ex

```diff
@@ -31,6 +35,7 @@ defmodule Hexpm.Accounts.Organizations do
       |> Multi.insert(:repository, fn %{organization: organization} ->
         %Repository{name: organization.name, public: false, organization_id: organization.id}
       end)
+      |> Multi.insert(:user, &User.build_organization(&1.organization))
```

Comment:
this step can fail with `{:error, :user, changeset,  _}` which we're not handling below. I was able to trigger it by attempting to create an organisation with a name of an already registered user:

<details>
<img width="1647" alt="image" src="https://user-images.githubusercontent.com/76071/58463851-e6aeab00-8134-11e9-9aa4-dc91d168758e.png">

</details>